packages/vuetify/src/components/VImg/VImg.ts

```diff
@@ -263,6 +270,8 @@ export default VResponsive.extend({
       }]
     }
 
+    node.data!.class = this.themeClasses
```

Comment:
This will make sure nobody messes this up by adding another `class = smth` line:
```suggestion
    node.data!.class = {
        ...node.data!.class,
        ...this.themeClasses,
    }
```

Reply:
or maybe use `mergeData` here?